docs/ai/build/rag.md

```diff
@@ -8,12 +8,29 @@ Retrieval Augmented Generation (RAG) allows developers to provide a knowledge ba
 
 Defining the RAG data set is largely up to the user to define. Currently only [Lance DB](https://lancedb.github.io/lancedb/) is supported. You can [review Lance DB's documentation](https://lancedb.github.io/lancedb/basic/) to determine the best way to ingest and embed your chosen RAG source data.
 
-We do provide an off the shelf way to create a table from markdown files. This will parse and chunk the content appropriately and use the `nomic-embed-text` model to generate vectors.
+We do provide a couple of tools to create a table from different sources.
+
+### From Markdown
+
+This will parse and chunk the content appropriately and use the `nomic-embed-text` model to generate vectors.
 
 ```shell
 subql-ai embed-mdx -i ./path/to/dir/with/markdown -o ./db --table your-table-name --model nomic-embed-text
 ```
 
+### From Web
+
+This will parse all the visible text from the specified web page(s). You can specify the scope for how many links are followed to pull in more data.
```

Comment:
How do we scrape these pages, it would be good to provide some details on the libarary we use. And I imagine there are some limitations on dynamic websites, e.g. does this work with websites that need to execute JS.

Finally, how can i verify if this was able to scrape my website, do we export the page content as text somewhere so i can verify this?